### PR TITLE
SHYRKA-1078-update-sdk-customAttrbutes-authenticateSession

### PR DESCRIPTION
**Pull request by ConorCrokeGenesys**

### CODE REVIEW

File: resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache [hunk 1]
```diff
@@ -216,6 +216,45 @@ public class WebMessagingClient {
         }
     }
 
+  /**
+   * Configures a new authenticated session with a randomly generated session token
+   *
+   * @param deploymentId     The ID of the Web Messaging deployment
+   * @param data             The session OAuthParams for configuring Authenticated Session
```
Comment (ConorCrokeGenesys) [inline resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache:223]: should @param origin be described here?

File: resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache [hunk 2]
```diff
@@ -216,6 +216,45 @@ public class WebMessagingClient {
...
+  public void configureAuthenticatedSession(String deploymentId, String origin, OAuthParams data) {
+    configureAuthenticatedSession(deploymentId, UUID.randomUUID().toString(), origin, data);
+  }
+
+  /**
+   * Configures an authenticated session using the provided session token. This can be used to reconnect to active sessions.
+   *
+   * @param deploymentId     The ID of the Web Messaging deployment
+   * @param token            The session token
+   * @param data             The session OAuthParams for configuring Authenticated Session
```
Comment (ConorCrokeGenesys) [inline resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache:234]: should @param origin be described here?

File: resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache (hunk 1 above)
Comment (ronanwatkins) [reply resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache:223]: Yep

File: resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache (hunk 2 above)
Comment (ronanwatkins) [reply resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache:234]: Yep

**Review by ronanwatkins — APPROVED** [review]

**Pull request merged**